### ISSUE TITLE
CRITICAL: aws_instance.root_block_device.volume_size not working in AWS "Govcloud" region

### ISSUE THREAD

**Issue by herbrandson** [terraform-providers/terraform-provider-aws#4469]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
Terraform v0.11.7
+ provider.aws v1.17.0

### Affected Resource(s)
* aws_instance

### Terraform Configuration Files

```hcl
provider "aws" {
  profile = "govcloud"
  region = "us-gov-west-1"
}

resource "aws_vpc" "default" {
    cidr_block = "10.10.0.0/16"
}

resource "aws_subnet" "default" {
    vpc_id = "${aws_vpc.default.id}"
    cidr_block = "10.10.0.0/20"
}

resource "aws_instance" "worker" {
    instance_type = "r4.xlarge"
    key_name = "liveearth"
    subnet_id = "${aws_subnet.default.id}"
    ami = "ami-e0aa3f81"

    root_block_device {
        volume_size = 50
    }
}
```

### Expected Behavior
This should create an instance with a root device of 50gb

### Actual Behavior
Terraform creates a new instance with 2 devices.
* /dev/xvda (8 gb), which is the root device
* /dev/xvdb (50 gb), which is not used

### Steps to Reproduce
1. `terraform apply`
2. Run `terraform apply` a second time and see that Terraform wants to delete and recreate the instance because the root device size is not correct

### Important Factoids
This is only happening when run in the AWS "Govcloud" region. The "East 1" regions works fine.

**Comment by herbrandson** [comment]
Sorry for the "CRITICAL" title, but this is causing huge issues for us right now. We currently can't run our Terraform scripts because it will delete and recreate all of our EC2 instances every time. We had scheduled some downtime last night and rebuilt our entire production cluster. Today, our servers have been crashing due to being out of disk space. I'm having to rebuild them manually one by one :(

Please help

**Comment by herbrandson** [comment]
Thanks so much. You guys are awesome!

**Comment by bflad** [comment]
This has been released in [version 1.18.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1180-may-10-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
